validation-test/compiler_crashers_2_fixed/issue61031.swift

```diff
@@ -0,0 +1,16 @@
+// RUN: %target-typecheck-verify-swift
```

Comment:
Do you mind sticking to the `issue-*` format for file names? This is the one I'm planning to use once I start renaming all the `sr*` files.